util/sokol_gl.h

```diff
@@ -3331,6 +3331,9 @@ static void _sgl_init_context(sgl_context ctx_id, const sgl_context_desc_t* in_d
     sg_pipeline_desc def_pip_desc;
     _sgl_clear(&def_pip_desc, sizeof(def_pip_desc));
     def_pip_desc.depth.write_enabled = true;
```

Comment:
It probably makes sense to remove this `depth.write_enabled = true` line when alpha-blending is enabled by default, and also remove this code in `_sgl_init_pipeline()` since it wouldn't have any purpose then:

https://github.com/floooh/sokol/blob/202f41b1ce892e1a3fcd25b5c9497a5871ac57a3/util/sokol_gl.h#L3146-L3148

The rest of the default pipeline state is configured for 2D rendering anyway, which IMHO makes it ok to switch the default pipeline state to alpha blended (e.g. the common setup for rendering alpha-blended 2D sprites).